### PR TITLE
Add missing jumps 

### PR DESCRIPTION
**Pull request by christianguenter2**
#868

Add Jump for DOCT, IARP, IASP, IATU, NROB, SFPF, SFPI, SMIM, SSST, STYL, TOBJ

### CODE REVIEW

**Comment by larshp** [comment]
wow, keeping busy? 👍

File: src/zabapgit_object_doct.prog.abap [hunk 1]
```diff
@@ -88,7 +88,43 @@ CLASS lcl_object_doct IMPLEMENTATION.
...
 
-    lcx_exception=>raise( 'todo, jump DOCT' ).
+    DATA: ls_dokentry TYPE dokentry,
+          ls_bcdata   TYPE bdcdata,
+          lt_bcdata   TYPE STANDARD TABLE OF bdcdata.
+
+    ls_dokentry-username = sy-uname.
+    ls_dokentry-langu    = sy-langu.
+    ls_dokentry-class    = c_id.
+    MODIFY dokentry FROM ls_dokentry.
```
Comment (larshp) [inline src/zabapgit_object_doct.prog.abap]: this seems bad, is there any SAP standard function we can call? if not please add a comment

Comment (christianguenter2) [inline src/zabapgit_object_doct.prog.abap]: AFAIK, no. Added an explaining comment.

**Comment by larshp** [comment]
perfect, merging, thanks 😄

**Pull request merged**